R-UST QoL tweaks

mikomyazaki opened the pull request:
:cl:
tweak: R-UST now has a prompt to tell you when hitting the shutdown button will cause an instant explosion, asking whether you're sure.
maptweak: R-UST room machinery starts the round anchored.
tweak: Improves the fuel injector controller UI, with injection rate and toggle all injectors controls.
/:cl:

Tweaks some simple quality of life things for the R-UST. 

Intending to use the R-UST for an away map, so... doing a bit of tweaking first to make it more usable as a main power source. This PR contains the non-controversial changes I want to make -- More to come with changing instability rates, fuels and power outputs in a future PR that will require a bit more time + testing.

mikomyazaki committed:
Adds a prompt to inform the user that shutting down the RUST when it is too hot will cause an explosion.

afterthought2 commented on code/modules/power/fusion/gyrotron/gyrotron.dm:
```diff
@@ -15,10 +15,10 @@
 /obj/machinery/power/emitter/gyrotron/anchored
 	anchored = 1
 	state = 2
+	construct_state = null
```
This is not the correct way of addressing the issue; set `base_type` instead.

mikomyazaki replied:
Done.

mikomyazaki committed:
Machinery in the R-UST room starts off anchored.

mikomyazaki commented:
Merge conflicts dealt with.

comma commented on nano/templates/fusion_injector_control.tmpl:
```diff
@@ -7,12 +15,20 @@
 		<div class="itemContent">
 			 {{if value.injecting}}
-				Online.<br/>{{:helper.link('Shut down.', null, {'toggle_injecting': value.ref})}}
+				Online.<br/>{{:helper.link('Shut down.', null, {'machine': value.ref, 'toggle_injecting': value.ref})}}
```
I'm confused, why are you adding this machine arg here, ref of machine is already passed in toggle_injecting

mikomyazaki replied:
Fix'd.

comma commented on nano/templates/fusion_injector_control.tmpl:
```diff
@@ -7,12 +15,20 @@
+				Online.<br/>{{:helper.link('Shut down.', null, {'machine': value.ref, 'toggle_injecting': value.ref})}}
 			 {{else}}
-				Offline.<br/>{{:helper.link('Start up.', null, {'toggle_injecting': value.ref})}}
+				Offline.<br/>{{:helper.link('Start up.', null, {'machine': value.ref,'toggle_injecting': value.ref})}}
```
ditto

mikomyazaki replied:
ditto

mikomyazaki committed:
Improves fuel injector UI + adds a toggle all injectors button.

Gives fusion injectors a variable injection rate controllable via the fusion injector control computer.
